src/cgame/cg_gameinfo.cpp

```diff
@@ -41,11 +41,11 @@ CG_ParseInfos
 int CG_ParseInfos( const char *buf, int max, char *infos[] )
 {
 	char *token;
-	int  count;
+	int  count = 0;
 	char key[ MAX_TOKEN_CHARS ];
 	char info[ MAX_INFO_STRING ];
 
-	count = 0;
+	int infoPostfixLen = strlen( "\\num\\" ) + strlen( va( "%d", MAX_ARENAS ) );
```

Comment:
Nullifying memory during allocation in `BG_Alloc` is making its usage easier.

But isn't it a performance loss to do it for strings or some structures, as some 
usages are shown? 
`BG_strdup` is clear example of inefficient usage.

May be that is better to change `BG_Alloc` contract and do null memory on
allocation only if needed?